build/rollup.config.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA ;
+ */
+
+import path from 'path';
+import glob from 'glob';
+import alias from 'rollup-plugin-alias';
+import handlebarsPlugin from 'rollup-plugin-handlebars-plus';
+import cssResolve from './css-resolve';
+import externalAlias from './external-alias';
+import resolve from 'rollup-plugin-node-resolve';
+
+const { srcDir, outputDir, aliases } = require('./path');
+const Handlebars = require('@oat-sa/tao-core-libs/src/handlebars');
+
+/**
+ * Support of handlebars 1.3.0
+ */
+const originalVisitor = Handlebars.Visitor;
+Handlebars.Visitor = function() {
+    return originalVisitor.call(this);
+};
+Handlebars.Visitor.prototype = Object.create(originalVisitor.prototype);
+Handlebars.Visitor.prototype.accept = function() {
+    try {
```

Comment:
I was sure that you will point this solution, because I know it looks ugly, but it is mandatory, because of `handlebars@1.3.0`.
Handlebars functions:

- `compile`: compiles template
- `accept`: resolves partials and helpers

Before version 3, handlebars `compile` function calls `accept` function, after some parsing. 
But from version 3, handlebars `accept` should be called before `compile`.

When `rollup-plugin-handlebars-plus` was created, it was made to support `v4.0.5`. So it calls `accept` before `compile`, but in `v1.3.0`, it fails, because some things are not ready.
So what I did, I kept the original `accept` function, because it will be called by `compile`, but I added a `try/catch`, to avoid `uncaughtException`, because rollup plugins calls it too early.
As we will update `handlebars` at least to `v3.0.0`, these lines should be removed.

And no, it won't be part of the rollup handlebars plugin. What I did there, and it was merged yesterday, is a support of custom `handlebars` version. So now, we can provide any version of handlebars compiler/runtime to rollup plugin.

Reply:
Ok, could you please just add a small comment like : 
`//TODO remove once migrated to hbs >= 3.0.0`